scipy/stats/_stats_py.py

```diff
@@ -1089,6 +1090,31 @@ def moment(a, order=1, axis=0, nan_policy='propagate', *, center=None):
         return _moment(a, order, axis, mean=center)
 
 
+def _demean(a, mean, axis, xp):
```

Comment:
A short comment or even docstring here would be nice

Reply:
Sure.
You noticed that this was just moved into a function from `_moment`, right? I meant to comment about it.